Migrate components and enable paths for Static layout

wackerow opened the pull request:
## Description
Migrates the following components used in Static layout markdown pages, and adds to the list of Static-layout-only components:
- `Contributors.tsx`
- `Logo.tsx`
- `MatomoOptOut.tsx`
- `SocialListItem.tsx`
- `TranslationChartImage.tsx`

Whitelists all markdown files using the static layout

<details>
<summary>

## English paths enabled

</summary>

- [community/get-involved](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/community/get-involved) ([Canonical](https://ethereum.org/en/community/get-involved))
- [community/grants](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/community/grants) ([Canonical](https://ethereum.org/en/community/grants))
- [community/language-resources](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/community/language-resources) ([Canonical](https://ethereum.org/en/community/language-resources))
- [community/online](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/community/online) ([Canonical](https://ethereum.org/en/community/online))
- [community/research](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/community/research) ([Canonical](https://ethereum.org/en/community/research))
- [contributing](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing) ([Canonical](https://ethereum.org/en/contributing))
- [contributing/adding-desci-projects](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-desci-projects) ([Canonical](https://ethereum.org/en/contributing/adding-desci-projects))
- [contributing/adding-developer-tools](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-developer-tools) ([Canonical](https://ethereum.org/en/contributing/adding-developer-tools))
- [contributing/adding-exchanges](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-exchanges) ([Canonical](https://ethereum.org/en/contributing/adding-exchanges))
- [contributing/adding-glossary-terms](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-glossary-terms) ([Canonical](https://ethereum.org/en/contributing/adding-glossary-terms))
- [contributing/adding-layer-2s](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-layer-2s) ([Canonical](https://ethereum.org/en/contributing/adding-layer-2s))
- [contributing/adding-products](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-products) ([Canonical](https://ethereum.org/en/contributing/adding-products))
- [contributing/adding-staking-products](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-staking-products) ([Canonical](https://ethereum.org/en/contributing/adding-staking-products))
- [contributing/adding-wallets](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/adding-wallets) ([Canonical](https://ethereum.org/en/contributing/adding-wallets))
- [contributing/content-resources](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/content-resources) ([Canonical](https://ethereum.org/en/contributing/content-resources))
- [contributing/design](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/design) ([Canonical](https://ethereum.org/en/contributing/design))
- [contributing/design/adding-design-resources](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/design/adding-design-resources) ([Canonical](https://ethereum.org/en/contributing/design/adding-design-resources))
- [contributing/design-principles](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/design-principles) ([Canonical](https://ethereum.org/en/contributing/design-principles))
- [contributing/quizzes](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/quizzes) ([Canonical](https://ethereum.org/en/contributing/quizzes))
- [contributing/style-guide](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/style-guide) ([Canonical](https://ethereum.org/en/contributing/style-guide))
- [contributing/style-guide/content-standardization](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/style-guide/content-standardization) ([Canonical](https://ethereum.org/en/contributing/style-guide/content-standardization))
- [contributing/translation-program](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program) ([Canonical](https://ethereum.org/en/contributing/translation-program))
- [contributing/translation-program/content-buckets](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/content-buckets) ([Canonical](https://ethereum.org/en/contributing/translation-program/content-buckets))
- [contributing/translation-program/faq](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/faq) ([Canonical](https://ethereum.org/en/contributing/translation-program/faq))
- [contributing/translation-program/how-to-translate](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/how-to-translate) ([Canonical](https://ethereum.org/en/contributing/translation-program/how-to-translate))
- [contributing/translation-program/mission-and-vision](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/mission-and-vision) ([Canonical](https://ethereum.org/en/contributing/translation-program/mission-and-vision))
- [contributing/translation-program/playbook](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/playbook) ([Canonical](https://ethereum.org/en/contributing/translation-program/playbook))
- [contributing/translation-program/resources](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/resources) ([Canonical](https://ethereum.org/en/contributing/translation-program/resources))
- [contributing/translation-program/translatathon](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/translatathon) ([Canonical](https://ethereum.org/en/contributing/translation-program/translatathon))
- [contributing/translation-program/translators-guide](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/contributing/translation-program/translators-guide) ([Canonical](https://ethereum.org/en/contributing/translation-program/translators-guide))
- [cookie-policy](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/cookie-policy) ([Canonical](https://ethereum.org/en/cookie-policy))
- [deprecated-software](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/deprecated-software) ([Canonical](https://ethereum.org/en/deprecated-software))
- [eips](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/eips) ([Canonical](https://ethereum.org/en/eips))
- [enterprise](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/enterprise) ([Canonical](https://ethereum.org/en/enterprise))
- [enterprise/private-ethereum](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/enterprise/private-ethereum) ([Canonical](https://ethereum.org/en/enterprise/private-ethereum))
- [foundation](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/foundation) ([Canonical](https://ethereum.org/en/foundation))
- [guides](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/guides) ([Canonical](https://ethereum.org/en/guides))
- [guides/how-to-create-an-ethereum-account](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/guides/how-to-create-an-ethereum-account) ([Canonical](https://ethereum.org/en/guides/how-to-create-an-ethereum-account))
- [guides/how-to-id-scam-tokens](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/guides/how-to-id-scam-tokens) ([Canonical](https://ethereum.org/en/guides/how-to-id-scam-tokens))
- [guides/how-to-revoke-token-access](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/guides/how-to-revoke-token-access) ([Canonical](https://ethereum.org/en/guides/how-to-revoke-token-access))
- [guides/how-to-use-a-bridge](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/guides/how-to-use-a-bridge) ([Canonical](https://ethereum.org/en/guides/how-to-use-a-bridge))
- [guides/how-to-use-a-wallet](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/guides/how-to-use-a-wallet) ([Canonical](https://ethereum.org/en/guides/how-to-use-a-wallet))
- [privacy-policy](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/privacy-policy) ([Canonical](https://ethereum.org/en/privacy-policy))
- [security](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/security) ([Canonical](https://ethereum.org/en/security))
- [staking/dvt](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/staking/dvt) ([Canonical](https://ethereum.org/en/staking/dvt))
- [terms-of-use](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/terms-of-use) ([Canonical](https://ethereum.org/en/terms-of-use))
- [zero-knowledge-proofs](https://deploy-preview-82--ethereum-org-fork.netlify.app/en/zero-knowledge-proofs) ([Canonical](https://ethereum.org/en/zero-knowledge-proofs))

</details>

## Related issue
https://www.notion.so/efdn/1st-wave-of-content-pages-migration-static-layout-d8fd3214d9ee4c1e9edaba58e89dc389?pvs=4

wackerow committed:
whitelist remaining static layout paths

wackerow committed:
implement getLastTagDate using git log

deprecate getLastDeployDate GitHub API fetch

wackerow committed:
migrate remaining components for Static layout

nhsz committed:
fix: /contributing/translation-program images path

wackerow committed:
rm unused export

wackerow committed:
fix unique key

switch to .login which should always be unique

wackerow committed:
Merge branch 'static-components' into static-layout-paths

wackerow committed:
Revert "Merge branch 'static-components' into static-layout-paths"

This reverts commit e1c4cbe07ce572257bf5e5cbc52a2bcee6079dfb, reversing
changes made to 7a28d52626e14a2d269b37a4f828a941210488e0.

wackerow committed:
Merge pull request #83 from ethereum/static-layout-paths

Static layout paths

wackerow committed:
move Callout to Static layout

pettinarip commented on src/components/MdComponents.tsx:
```diff
@@ -253,24 +252,22 @@ export const reactComponents = {
-  // Contributors,
-  // Logo,
-  // MeetupList,
+  Contributors,
```
Can't ignore that this one is not ordered alphabetically 😆

wackerow replied:
🤣

pettinarip reviewed (commented):
Looks good @wackerow

It seems that we need to update this branch and fix some conflicts. There are a few changes that I think are coming from an older branch.

wackerow committed:
sort components alphabetically

wackerow committed:
Merge branch 'dev' into static-components

pettinarip commented on public/content/translations/id/contributing/translation-program/how-to-translate/index.md:
Why do we need to move these files? was this an issue in the fork update? or is something coming from the main repo?

wackerow replied:
Gatsby repo, searching for the image in question:
```
5 results - 5 files

src/content/contributing/translation-program/how-to-translate/index.md:
  38  
  39  
  40: ![Translated and untranslated files in Crowdin](./crowdin-files.png)
  40  
  41  

src/content/translations/id/contributing/translation-program/index.md:
  59  
  60:    ![File yang diterjemahkan dan tidak diterjemahkan di Crowdin](./crowdin-files.png)
  61  

src/content/translations/pt-br/contributing/translation-program/how-to-translate/index.md:
  37  
  38  
  39: ![Arquivos traduzidos e não traduzidos no Crowdin](./crowdin-files.png)
  39  
  40  

src/content/translations/sl/contributing/translation-program/index.md:
  51  
  52  
  53:    ![Prevedene in neprevedene datoteke na Crowdinu](../../../../contributing/translation-program/crowdin-files.png)
  53  
  54  

src/content/translations/tr/contributing/translation-program/index.md:
  66  
  67:    ![Crowdin'deki çevrilmiş ve çevrilmemiş dosyalar](./crowdin-files.png)
  68  
```

It would appear that the English page had a section for "how to translate" which has since been pulled out in a more recent update, but this has only been updated for `pt-br`, not for `id,sl,tr`.

I think we should pull these back to their previous paths, update the relative paths to point to the image in the english directory, and await later updates to these pages from Crowdin imports.

wackerow committed:
update comment counter

nhsz committed:
Merge branch 'dev' into static-components

wackerow committed:
Merge branch 'static-components' of https://github.com/ethereum/ethereum-org-fork into static-components

wackerow committed:
Merge branch 'dev' into static-components

wackerow committed:
[debug] disable translation-program subpaths

wackerow committed:
fix image paths, re-enable pages

reverts moving sl/id/tr translation-program/index into how-to-translate subpath

wackerow commented on src/lib/utils/md.ts:
```diff
@@ -15,15 +15,30 @@ const getPostSlugs = (dir: string, files: string[] = []) => {
-    "/community/code-of-conduct",
-    "/community/events",
-    "/community/support",
+    // Use cases (7/7) ✅
```
We certainly don't need to keep these comments, but I found them a little helpful to just keep track of all of these... figured this will be removed entirely very soon with the dev docs

pettinarip commented on src/components/Contributors.tsx:
```diff
@@ -1,14 +1,12 @@
 
-export interface IProps {}
+// TODO: Re-import once Crowdin scripts are functional and data available
+// import data from "../data/contributors.json"
```
👌🏼

pettinarip reviewed (approved):
Nice job @wackerow 

Not related with this PR but any idea why the ToC is not working in this page for example?
- preview: https://deploy-preview-82--ethereum-org-fork.netlify.app/community/online
- prod: https://ethereum.org/community/online/
